pkg/gui/controllers/branches_controller.go

```diff
@@ -117,6 +117,12 @@ func (self *BranchesController) GetKeybindings(opts types.KeybindingsOpts) []*ty
 			Tooltip:           self.c.Tr.MergeBranchTooltip,
 			DisplayOnScreen:   true,
 		},
+		{
+			Key:               opts.GetKey(opts.Config.Branches.SquashBranch),
+			Handler:           opts.Guards.OutsideFilterMode(self.squash),
+			GetDisabledReason: self.require(self.singleItemSelected(self.notMergingIntoYourself)),
+			Description:       self.c.Tr.Squash,
```

Comment:
Let's add `OpensMenu: true` to this so that the user can see that the keybinding opens a menu